File: 11-09-2023 1074/1074.c
#include <stdio.h>
 
int main() {
 
    int valor, contador;
    int i;

    scanf("%d", &contador);

    for (i = 0; i < contador; i++){
        scanf("%d", &valor);

        if(valor == 0){
            printf("NULL\n");
        } else {
            if (valor > 0 && valor % 2 == 0){
                printf("EVEN POSITIVE\n");
            }
            if (valor < 0 && valor % 2 == 0){
            printf("EVEN NEGATIVE\n"); 
            }
            if (valor > 0 && valor % 2 != 0){
            printf("ODD POSITIVE\n"); 
            }
            if (valor < 0 && valor % 2 != 0){
            printf("ODD NEGATIVE\n"); 
            }
        }
    }
 
    return 0;
}

File: Iniciante/Pagina 3/1074/1074.py
casos = int(input())
i = 0

while i < casos:
    valor = int(input())

    if valor == 0:
        print("NULL")
    else:
        if valor > 0 and valor % 2 == 0:
            print("EVEN POSITIVE")
        if valor < 0 and valor % 2 == 0:
            print("EVEN NEGATIVE")
        if valor > 0 and valor % 2 != 0:
            print("ODD POSITIVE")
        if valor < 0 and valor % 2 != 0:
            print("ODD NEGATIVE")
    
    i += 1;